client/src/App/index.js

```diff
@@ -1,23 +1,17 @@
 import React from 'react';
-import logo from '../logo.svg';
 import './App.css';
+import { Switch, Route } from 'react-router-dom';
+import LoginPage from '../pages/logIn';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <Switch>
+          <Route exact path="/login">
```

Comment:
```suggestion
          <Route exact path="/dashboard/login">
```

Reply:
but it is a free route and the dashboard is supposed to be protected?